src/edwards.rs

```diff
@@ -1289,6 +1327,20 @@ mod test {
         0x2b, 0x42,
     ]);
 
+    /// The largest valid scalar (not mod l). Remember for NAF computations, the top bit has to be
+    // 0. So the largest integer a scalar can hold is 2^255 - 1. Addition and subtraction are
+    // broken on unreduced scalars. The only thing you can do with this is multiplying with a curve
+    // point (and actually also scalar-scalar multiplication, but that's just a quirk of our
+    // implementation).
+    #[cfg(feature = "precomputed-tables")]
+    static LARGEST_UNREDUCED_SCALAR: Scalar = Scalar {
```

Comment:
Why is this defined both here and in scalar.rs?

Reply:
Will fix. See below